src/procedures/InvestInSimpleSto.ts

```diff
@@ -14,23 +14,45 @@ import { SecurityToken, SimpleSto } from '../entities';
 import { ApproveErc20 } from './ApproveErc20';
 import { Factories } from '../Context';
 
+/**
+ * @hidden
+ */
 export const createRefreshSecurityTokenFactoryResolver = (
   factories: Factories,
   securityTokenId: string
 ) => async () => {
   return factories.securityTokenFactory.refresh(securityTokenId);
 };
 
+/**
+ * @hidden
+ */
 export const createRefreshSimpleStoFactoryResolver = (
   factories: Factories,
   simpleStoId: string
 ) => async () => {
   return factories.simpleStoFactory.refresh(simpleStoId);
 };
 
+/**
+ * Procedure that allows invest in the STO with Ether
+ */
 export class InvestInSimpleSto extends Procedure<InvestInSimpleStoProcedureArgs> {
   public type = ProcedureType.InvestInSimpleSto;
 
+  /**
+   * - Buy tokens with Ether
+   *
+   * Note that this procedure will fail if:
+   * - The security token symbol doesn't exist
+   * - The STO address is invalid
+   * - The STO is either archived or hasn't been launched
+   * - The STO hasn't started yet
+   * - The STO is paused
+   * - The STO has already been finalized
+   * - The STO doesn't allow beneficial investments
```

Comment:
`- Attempting to invest on behalf of another address when beneficial investments aren't allowed for the STO`